lib/workers/branch/automerge.ts

```diff
@@ -27,7 +27,7 @@ export async function tryBranchAutomerge(
   }
   const branchStatus = await platform.getBranchStatus(
     config.branchName,
```

Comment:
```suggestion
  const branchStatus = config.ignoreTests ? BranchStatus.green : await platform.getBranchStatus(config.branchName
```

I thought about this, needs prettier run

Reply:
Do we need to add logging?

Reply:
🤔 Makes sense, so we shoud wrap `config.ignoreTests ? BranchStatus.green : await platform.getBranchStatus(config.branchName)` to a new function at worker util and add logging there.

Reply:
Thank you. Now it is much clearer. As you can see it requires a lot of changes, also will need to align unit tests. Can we create issue for this changes and assign to me?